src/icalendar/prop/__init__.py

```diff
@@ -101,6 +113,29 @@ def __eq__(self, other):
         """self == other"""
         return isinstance(other, vBinary) and self.obj == other.obj
 
+    @classmethod
+    def examples(cls) -> list[vBinary]:
+        """Examples of vBinary."""
+        return [cls("VGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4")]
+
+    from icalendar.param import VALUE
+
+    def to_jcal(self, name: str) -> list:
+        """The jcal represenation of this property according to :rfc:`7265`."""
+        params = self.params.to_jcal()
+        if params.get("encoding") == "BASE64":
+            # BASE64 is the only allowed encoding
+            del params["encoding"]
+        return [name, params, self.VALUE.lower(), self.obj]
+
+    @classmethod
+    def from_jcal(cls, ical_property: list) -> vBinary:
+        """Parse jcal from :rfc:`7265` to a vBinary."""
+        return cls(
```

Comment:
More validation and raise `JCalParsingError`?

Reply:
vBinary is validated.